pkgs/build-support/vm/default.nix

```diff
@@ -322,7 +322,6 @@ rec {
      that allows you to boot into the VM and debug it interactively. */
 
   runInLinuxVM = drv: lib.overrideDerivation drv ({ memSize ? 512, QEMU_OPTS ? "", args, builder, ... }: {
-    requiredSystemFeatures = [ "kvm" ];
```

Comment:
I would be supportive of a change that makes KVM optional but still informs the remote build scheduling correctly. Such an attribute would have to be implemented in nix itself.
 - https://github.com/NixOS/nix/issues/8316

Note that this line of code mostly affects fairly simple VM-based builds, such as some builders that create an image with a filesystem on it. It does not affect the NixOS VM tests.